src/hydrateSatelliteObject.js

```diff
@@ -65,38 +65,39 @@ module.exports = function(buildInfo, setDebugOutputEnabled) {
    * will be stored for the session only.
    */
   _satellite.setCookie = function(name, value, days) {
-    logger.warn('_satellite.setCookie is deprecated. Please use the `_satellite.cookie` API.');
-
+    var optionsStr = '';
     var options = {};
 
     if (days) {
-      var date = new Date();
-      date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
-      options.expires = date;
+      optionsStr = ', { expires: ' + days + ' }';
+      options.expires = days;
     }
 
-    document.cookie = cookie.serialize(name, value, options);
+    var msg = '_satellite.setCookie is deprecated. Please use ' +
+      '_satellite.cookie.set("' + name + '", "' + value + '"' + optionsStr + ').';
+
+    logger.warn(msg);
+    _satellite.cookie.set(name, value, options);
   };
 
   /**
    * Reads a cookie value.
    * @param {string} name The name of the cookie to read.
    * @returns {string}
    */
-  _satellite.getCookie = _satellite.readCookie = function(name) {
```

Comment:
I added this `getCookie` alias a while back I believe before we deprecated `_satellite.readCookie`. It doesn't actually exist in DTM. Now that `_satellite.readCookie` is deprecated, there's no real need for the alias.